File: tests/fixtures/csharp/skips_and_vacuous.cs
using System;
using Xunit;

namespace Discipline.Tests
{
    public class SkipsAndVacuousTest
    {
        [Fact(Skip = "Temporarily disabled in CI")]
        public void TestSkippedFact()
        {
            Assert.Equal(1, 1);
        }

        [Fact]
        public void TestTautology()
        {
            Assert.True(true);
        }

        [Fact]
        public void TestEmptyMethod()
        {
            // Empty method with zero assertions
        }
    }
}

File: tests/fixtures/ruby/skips_and_vacuous.rb
require 'minitest/autorun'

class SkipsAndVacuousTest < Minitest::Test
  def test_skipped_suite
    skip "Temporarily skipped in CI"
    assert_equal 1, 1
  end

  def test_tautology
    assert true
  end

  def test_empty
    # Empty test method with zero assertions
  end
end

RSpec.describe "DisabledFeatures" do
  xit "is pending implementation" do
    expect(true).to eq(true)
  end
end
